src/components/MentorsReport/MentorsReport.js

```diff
@@ -135,87 +129,74 @@ class ToPrint extends Component {
-            </div>
+          <Grid item xs={12} sm={12} md={12} lg={12}>
+            <Typography variant="h6">{name}</Typography>
+            <Typography>{specialty}</Typography>
+            <Typography>
+              {location} , {stateCode}
+            </Typography>
           </Grid>
         </Grid>
-        <div>
-          <Typography variant="body2" className={classes.sectionMargin}>
-            Describe the activities you completed as a mentor in the FLAD
-            Mentorship program.
-          </Typography>
-          <FormControl required className={classes.formControl}>
-            <TextField
-              id="activitiesDesc"
-              name="activitiesDesc"
-              multiline
-              rows="8"
-              fullWidth
-              value={activitiesDesc}
-              onChange={handleChange}
-              placeholder="Your report here"
-              required
-              onBlur={checkForNull}
-              variant="outlined"
-              autoFocus
-            />
-          </FormControl>
-          <FormHelperText error={true}>{activitiesDescError}</FormHelperText>
-        </div>
-
-        <div className={classes.chipContainer}>
-          <FormControl className={classes.formControl}>
-            <InputLabel htmlFor="select-multiple-chip">
-              Mentees helped
-            </InputLabel>
-            <Select
-              multiple
-              value={selectedMentees}
-              name="selectedMentees"
-              onChange={handleChange}
-              input={<Input id="select-multiple-chip" />}
-              renderValue={selected => (
-                <div className={classes.chips}>
-                  {selected.map(value => (
-                    <Chip key={value} label={value} className={classes.chip} />
-                  ))}
-                </div>
-              )}
-              MenuProps={MenuProps}
-            >
-              {mentees &&
-                mentees.map(mentee => (
-                  <MenuItem key={mentee} value={mentee}>
-                    <Checkbox checked={selectedMentees.indexOf(mentee) > -1} />
-                    <ListItemText primary={mentee} />
-                  </MenuItem>
+        <Typography variant="body2" className={classes.sectionMargin}>
+          Describe the activities you completed as a mentor in the FLAD
+          Mentorship program.
+        </Typography>
+        <FormControl required className={classes.formControl}>
+          <TextField
+            id="activitiesDesc"
+            name="activitiesDesc"
+            multiline
+            rows="8"
+            fullWidth
+            value={activitiesDesc}
+            onChange={handleChange}
+            placeholder="Your report here"
+            required
+            onBlur={checkForNull}
+            variant="outlined"
+            autoFocus
+          />
+        </FormControl>
+        <FormHelperText error={true}>{activitiesDescError}</FormHelperText>
```

Comment:
I think the error text "Please introduce a activitiesDesc" will be difficult to understand for the users. Can we use a more descriptive text for this error?

![screenshot 2019-01-24 at 14 06 31](https://user-images.githubusercontent.com/19712816/51680687-bddda400-1fe2-11e9-8b31-6c845714abc9.png)

Reply:
I completely forgot about this. Thanks for pointing this out. I have changed the error message for something more descriptive.